node/clients/search.ts

```diff
@@ -207,7 +215,12 @@ export class Search extends JanusClient {
           sku: skuId,
           refid,
           sellers: res.data.SkuSellers.filter((item: any) => {
-            return item.IsActive === true
+            // check if seller is available in current sales channel
+            const inSellersList = this.sellersList?.find(seller => {
```

Comment:
It would be more optimal to have a set and search for sellerId there versus using a .find

Reply:
Done